src/protocol.cpp

```diff
@@ -108,15 +108,15 @@ const static std::vector<std::string> tiertwoNetMessageTypesVec(std::find(allNet
 
 CMessageHeader::CMessageHeader(const MessageStartChars& pchMessageStartIn)
 {
-    memcpy(pchMessageStart, pchMessageStartIn, MESSAGE_START_SIZE);
+    memcpy(pchMessageStart, pchMessageStartIn, CMessageHeader::MESSAGE_START_SIZE);
```

Comment:
No need to update for `CMessageHeader`'s methods.

Reply:
updated